assets/css/src/generic/_typography.scss

```diff
@@ -15,3 +15,25 @@ a {
     text-decoration: none;
   }
 }
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  color: $color-text;
+  font-weight: bold;
+  line-height: 1.25;
+  margin: 0 0 20px 0;
```

Comment:
Our colours definitions are in the settings file. Should we do the same with dimensions? I like how gutenberg does that: https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_variables.scss